runtime/near-vm-runner/RUNTIMES.md

```diff
@@ -0,0 +1,218 @@
+This document describes the considerations that go into picking a Wasm runtime as well as the
+conclusions we made after evaluating a number of VM backends and frontends.
+
+# Criteria
+
+A number criteria have been used to evaluate a Wasm runtime for use in the implementation of the
+NEAR protocol. Some of the criteria are already listed in the [FAQ] document and this document
+gives a more thorough look. Listed roughly in the order of importance:
+
+* security – how well does the runtime deal with untrusted input;
+* correctness – does the runtime do what it claims to do;
+* reliability – how much confidence is there in the implementation of runtime;
+* platform support – does the runtime support targets that we want to target;
```

Comment:
Guess this sums up to "Linux/x86 for development and deployment and macOS arm/x64 for development".

Reply:
Correct, this is also described in the Plaform Support section within Requirements.